backend/plugins/mongolian_api/src/modules/erkhet/graphql/resolvers/syncLog.ts

```
@@ -0,0 +1,83 @@
+      subdomain,
+      pluginName: 'core',
+      module: 'users',
+      action: 'findOne',
+      method: 'query',
+      input: { _id: syncLog.createdBy },
+      defaultValue: {},
+    });
+
+  },
+
+  async content(syncLog: ISyncLogDocument, _, {}: IContext) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)